### ISSUE TITLE
How to Accept Payments with React and Stripe

### ISSUE THREAD

**Issue by gustavoazevedoo** [gustavoazevedoo/github-blog#2]
Payments are an essential part of any online business. But the process of setting up those payments can be incredibly complex at times.

To accept payments, developers were traditionally required to set up code both on the client and server. This was in addition to learning complex third-party APIs which required parsing through large amounts of documentation.

Fortunately tools like Stripe Checkout make handling purchases on the web and mobile devices easier than ever before.

In this tutorial, you'll take a look at how to set up Stripe Checkout to accept credit card, Apple Pay, and Google Pay payments in our React apps.

Want a great resource to learn React? Become a React pro in 30 minutes a day with my [React Bootcamp](https://reactbootcamp.com/).

## What is Stripe Checkout?

![image](https://user-images.githubusercontent.com/26803662/191437769-1cda0068-b8fd-4c00-9a40-04207a037557.png)

Stripe Checkout is an all-in-one tool that not only streamlines and simplifies the process of checking out for us as developers. It also gives us a user interface for our customers to use that's optimized for performance and usability.

When using Stripe checkout over alternative options such as Stripe Elements, the benefit is that you have to write far less code to achieve the same end result. Additionally, you get features such as automatic translations for global users in their language.

You also get the ability to customize this user interface without having to write a single React component.

## Stripe Checkout is Hosted on Stripe
Be aware that Stripe Checkout is hosted on Stripe's servers and it involves taking the user to checkout.stripe.com. This, however, shouldn't be viewed as a downside.

Because trust is such an important part of the checkout process, it's often a large benefit to know that Stripe is handling the payment. As a result, customers can be far more secure and confident in the payment process.

In short, it's a wise decision to use a pre-made solution such as Stripe Checkout, considering it will not only mean less work for you, but will also create higher trust in your product.

ps: Trecho de um post do [Reed Barger](https://www.freecodecamp.org/news/author/reed/) 

React developer who loves to make incredible apps. Showing you how at ReactBootcamp.com 

 [Post completo aqui.](https://www.freecodecamp.org/news/react-stripe-payments/)
